owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.java

```diff
@@ -69,6 +69,8 @@ public class FolderPickerActivity extends FileActivity implements FileFragment.C
 
     private static final String TAG_LIST_OF_FOLDERS = "LIST_OF_FOLDERS";
 
+    private static final String ACTION_BUTTON = "ACTION_BUTTON";
```

Comment:
Let's add the EXTRA prefix, make it public(so other classes can use it) and rename it according to the enum class name 👍

Reply:
Done